common/display/display.cpp

```diff
@@ -164,8 +164,18 @@ bool Display::GetActiveConfig(uint32_t *config) {
   return true;
 }
 
-bool Display::SetDpmsMode(uint32_t dpms_mode) {
-  return display_queue_->SetDpmsMode(dpms_mode);
```

Comment:
May be try by reverting this commit https://github.com/android-ia/frameworks-base/commit/ed03ff5defcbcb1b07a72a3c6c92a208aab19a1d